cucumber/step_definitions/careerPage.js

```diff
@@ -1,5 +1,110 @@
-'use strict';
-
-const { Given, When, Then, setDefaultTimeout } = require('cucumber');
+"use strict";
 
```

Comment:
When you will have a big step definition file with 40-50 step definitions, it will be more difficult to find a step definition. A best practice is that you separate the When, Then steps, so the When steps are at the top of the file and the Then steps at the bottom of the file. Could you please order them?

Reply:
done